### PR TITLE
added favourites page

### LINKED ISSUE #3: Add boilerplate HTML for favourites page
-add boilerplate html for favourites page
- add .js and .css file
- add images folder
- link .js and .css in html

### PR DESCRIPTION
**Pull request by PDUBB3**
- Created Html for favourites page
- Created CSS for favourites page
- Created Javascript
- Linked Jquery

### CODE REVIEW

File: favourites.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/css/favourites.css" />
+    <title>Travel Companion</title>
```
Comment (surajverma2587) [inline favourites.html]: Is that suppose to say Travel Companion?

Comment (PDUBB3) [reply favourites.html]: addressed in this commit :) 
2e4127c8ce3a2b66cf01c8c2a90835ddc608e4ad

Comment (surajverma2587) [reply favourites.html]: Ok that looks good. Please keep that in mind the next time you do something like this. Possibly read the docs and best practices as well!!

Comment (surajverma2587) [reply favourites.html]: But I appreciate the effort :)

**Review by Cratesy — APPROVED** [review]
changes are good thanks

**Pull request merged**